### ISSUE TITLE
[JOSS review] import fails after installation via pip on mac os

### ISSUE THREAD

**Issue by janfb** [Saran-nns/sorn#25]
Hi there, as part of the review at https://github.com/openjournals/joss-reviews/issues/3545 (see also https://github.com/openjournals/joss-reviews/issues/3554) I was following the installation instructions like this: 
- create conda env with python 3.8
- install via `pip install sorn`
- in python shell run `import sorn`

and got the following error:
```
Python 3.8.10 (default, May 19 2021, 11:01:55)
[Clang 10.0.0 ] :: Anaconda, Inc. on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import sorn
Traceback (most recent call last):
  File "/Users/Jan/anaconda3/envs/sorn-review/lib/python3.8/site-packages/sorn/sorn.py", line 10, in <module>
    from sorn.utils import Initializer
  File "/Users/Jan/anaconda3/envs/sorn-review/lib/python3.8/site-packages/sorn/utils.py", line 9, in <module>
    import networkx as nx
ModuleNotFoundError: No module named 'networkx'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/Jan/anaconda3/envs/sorn-review/lib/python3.8/site-packages/sorn/__init__.py", line 1, in <module>
    from .sorn import Simulator, Trainer
  File "/Users/Jan/anaconda3/envs/sorn-review/lib/python3.8/site-packages/sorn/sorn.py", line 12, in <module>
    from utils import Initializer
ModuleNotFoundError: No module named 'utils'
```

Any suggestions for a fix? 

Best, 
Jan

**Comment by janfb** [comment]
the same problem occurs on Ubuntu 20.04

**Comment by Saran-nns** [comment]
Hi @janfb I checked the problem with the Windows and Ubuntu machines, the code works without problems up to py3.8 and for py3.9 there are version compatibility issues between dependencies. 
I have also checked the Mac OS virtual environments and there are no problems when the package is imported at Jupyter notebooks. 
However, when I import the package from the terminal, I got errors and was able to reproduce the same as you reported. I suspect that the problem might be related to the system 'PATH'. Thank you for reporting the problem. I am investigating it.
